src/libraries/System.Net.Http/tests/UnitTests/System.Net.Http.Unit.Tests.csproj

```diff
@@ -329,10 +329,10 @@
     <Compile Include="..\..\src\System\Net\Http\SocketsHttpHandler\SystemProxyInfo.cs">
       <Link>ProductionCode\System\Net\Http\SocketsHttpHandler\SystemProxyInfo.cs</Link>
     </Compile>
-    <Compile Include="..\..\src\System\Net\Http\SocketsHttpHandler\SystemProxyInfo.OSX.cs" Condition=" '$(TargetsOSX)' == 'true' and '$(TargetFramework)' == '$(NetCoreAppCurrent)'">
+    <Compile Include="..\..\src\System\Net\Http\SocketsHttpHandler\SystemProxyInfo.OSX.cs" Condition=" ('$(TargetsOSX)' == 'true' or '$(TargetsiOS)' == 'true') and '$(TargetFramework)' == '$(NetCoreAppCurrent)'">
```

Comment:
Nit: I know it was like that but it might be worth removing the spaces in between the condition and the surrounding quotes `"` in this file.

Reply:
Nevermind, there is a lot more files that do this. Maybe we can open an issue to cleanup.